web/src/features/fbaCalculator/FireBehaviourAdvisoryCalculatorPage.tsx

```diff
@@ -54,6 +54,10 @@ export const FireBehaviourAdvisoryCalculator: React.FunctionComponent = () => {
               of the Canadian Forest Fire Behaviour Prediction System&quot; from Forestry
               Canada Fire Danger Group, Information Report ST-X-3, 1992.
             </p>
+            <p>
+              If you have any questions or curiosities, send us an{' '}
+              <a href="mailto: bcws.predictiveservices@gov.bc.ca">email.</a>
```

Comment:
as an FYI, we've generally made it a convention that we automatically insert a subject line in the mailto link
(e.g.,  href={`mailto:bcws.predictiveservices@gov.bc.ca?subject=Predictive Services Unit - Fire Behaviour Advisory Calculator`} ) - see the `Contact.tsx` file under our general components subdir for more context.
Would be a handy little addition to this piece.